doc/conf.py

```diff
@@ -148,9 +148,9 @@ def linkify_issues_in_changelog(app, docname, source):
         # this path trickery is needed because this script can
         # be invoked with different working directories:
         # * running make in docs/
-        # * running python setup.py build_sphinx in the repo root dir
+        # * running tox -e docs in the repo root dir
 
-        with open(changelog_path) as f:
+        with open(changelog_path, encoding="utf-8") as f:
```

Comment:
Note: This is a bug fix of "our document". So we don't need to mention it on CHANGES.